src/data/blog.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "id": "liveness-2-and-beyond",
+    "name": "Liveness 2 and beyond",
+    "description": "",
```

Comment:
The current status of on-chain storage proofs

Reply:
done https://github.com/defi-wonderland/web/pull/57/commits/f199666272a4c20f6846129c7ee6802e7116c65f